css/styles.css

```diff
@@ -154,7 +154,7 @@ html::-webkit-scrollbar-thumb {
 
   /* ******************************** About Conference *************** */
   .about-conference {
```

Comment:
Please ensure that the pages look almost identical to the original design. And please ensure there are no images or elements blocking the text in both screen sizes.

Your Implementation | Figma Design
:---:|:---:
   
![overlap](https://user-images.githubusercontent.com/10332499/152071756-6a1a8194-e676-4db1-adb2-6c6c4c1f5b06.PNG)  | ![design](https://user-images.githubusercontent.com/10332499/152071786-a0412163-8349-4a63-8c12-d6067c25c9ad.PNG)